openff/toolkit/topology/molecule.py

```diff
@@ -3824,8 +3849,32 @@ def from_polymer_pdb(
             offmol.atoms[i].metadata["chain_id"] = atom.residue.chain.id
         offmol.add_default_hierarchy_schemes()
 
+        if len(offmol.chains) > 1:
+            raise MultipleMoleculesInPDBError(
+                "Multiple polymer chains were detected in the input. The OpenFF "
+                + "Toolkit requires that only one polymer chain is present in a "
+                + "PDB, and that it is the only molecule present. Try splitting "
+                + "each polymer chain into its own PDB with another tool, and "
+                + "import any small molecules with Topology.from_pdb_and_smiles."
+            )
```

Comment:
To the OpenFF toolkit, Chain IDs are just cosmetic. So it's fine if one Molecule has atoms with multiple chain IDs

```suggestion
```